ci/docs.yml

```diff
@@ -0,0 +1,28 @@
+name: uxarray-docs
+channels:
+    - conda-forge
+dependencies:
+    - python=3.9
+    - dask
+    - netcdf4
+    - numba
+    # pinning the numpy version for numba compatibility
+    - numpy>=1.2, <1.22
+    - pathlib
+    - pre_commit
+    - pytest
+    - pytest-cov
+    - scipy
+    - xarray
+    - pip
+    - ipykernel
+    - ipython
+    - jupyter_client
+    - matplotlib-base
+    - sphinx-book-theme
+    - myst-nb
+    - sphinx-design
+    - nbsphinx
+    - uxarray
```

Comment:
Should we assume from this `uxarray` dependency that the documentation will be generated only from the conda-latest version? If so, wouldn't that be an issue for testing PRs to see if they pass docs build?

Reply:
Right! Sorry I meant to bring this up in the PR or at our meeting.

I'm not entirely sure how we should handle this. My thoughts on the matter include:

- Docs should probably not include functionality that is not officially released, unless we want to implement some sort of "latest" and "stable" distinction on the docs (xarray does this)
- adding uxarray in the env file shouldn't actually affect this, since local generation looks for local files and RTD generation does similar the same after a clone, I just forgot to remove it after playing around with seeing if including it in the env file would have any effect 😅

Anyway, I've removed uxarray from the docs env file.

Reply:
@erogluorhan Actually, I've looked in to how RTD handles this, and I've gone ahead and changed the default version of the docs to point to stable instead of latest. 

I want to make sure it behaves as expected, but I think RTD will take care of updating stable to pull from the most recent tagged version.

Reply:
> - adding uxarray in the env file shouldn't actually affect this, since local generation looks for local files and RTD generation does similar after a clone, I just forgot to remove it after playing around with seeing if including it in the env file would have any effect 😅

 No worries, got it :)

> - Docs should probably not include functionality that is not officially released, unless we want to implement some sort of "latest" and "stable" distinction on the docs (xarray does this)

Agreed!

> I want to make sure it behaves as expected, but I think RTD will take care of updating stable to pull from the most recent tagged version.

This is great if it will happen! Let's make sure about this when we merge a PR and create a tag also.